main/integration-tests/__test__/api-tests/workspace-service-catalogs/create-workspace-service-catalog.test.js

```diff
@@ -166,4 +179,90 @@ describe('Create workspace-service-catalog scenarios', () => {
       });
     });
   });
+  describe('Creating a workspace service catalog environment with studies', () => {
+    it('for EC2Linux', async () => {
```

Comment:
usually for the `it()` part it should kinda read like a sentence that clearly describes what the test itself does--`describe()` says broadly what that test suite is for. So like for these, you could have like:

`describe('Workspace SC Env with studies, () => {
    it('should correctly create a workspace with studies on EC2Linux', ...`

something like that, or whatever your test is designed to do

Reply:
I see, I made a change accordingly to the test blocks.